x-pack/plugins/apm/public/hooks/use_fetcher.tsx

```diff
@@ -124,6 +125,10 @@ export function useFetcher<TReturn>(
       // This can happen if the data fetching is conditional (based on certain inputs).
       // In these cases it is not desirable to invoke the global loading spinner, or change the status to success
       if (!promise) {
+        setResult({
+          status: FETCH_STATUS.DROPPED,
+        });
```

Comment:
`DROPPED` sounds like it's permanently dropped but it could later change to `LOADING` and then `SUCCESS`/`FAILURE`, right? 
What's the difference between `DROPPED` and `NOT_INITIATED`?